designsafe/static/scripts/projects/components/edit-project/edit-project.component.html

```diff
@@ -162,7 +162,7 @@ <h4 class="modal-title" style="border:none;">
                                     type="text"
                                     id="id_pi"
                                     name="pi"
-                                    autocomplete="username"
+                                    autocomplete="chrome-off"
```

Comment:
Why chrome-off for some, and off for others?

Reply:
I've read that Chrome does not respond to `autocomplete="off"` while firefox does. Chrome does not seem to have any reliable way to disable autocomplete unless you create some sort of hacky workaround. I've added `autocomplete="chrome-off"` to these inputs which is what most people are having success with. As long as you have less than 5 of these `autocomplete="chrome-off"` inputs in a form, Chrome should respect the fields. If Chrome still ignores the autocomplete settings, we may have to come up with a more involved way to disable the autocomplete field.

Reply:
Gotcha, thanks for the explanation!